### DISCUSSION TITLE
Download Key missing - but it's not

### DISCUSSION THREAD

**Discussion by PaulSrch** [akeeba/panopticon#814] [Q&A (Community Support)]
I'm getting an error on one site that says "A download key is missing or invalid". The Joomla admin module says that all Download Keys are entered correctly, and I've double checked them. Aside from Akeeba Backup, the only other extension I have that uses a download key is Latest News Enhanced Pro, and I the key is correct. Any idea what might be happening and how to resolve it? And how I could determine _which_ key is the issue?

**Comment by nikosdion** [comment]
This message can never be shown in error. It communicates information sent by Joomla.

Kindly note that Joomla! will only display this information to you in two places. As a control panel quick icon shown always, taking you to the Update Sites page where missing download keys are reported. And as a notice when an update is found. You probably went to the Updates page which is where you last saw this, but since there was no update available for the extension missing the Download Key you didn't see anything.

You can very easily see this information in Panopticon, though. After all, it's a site management and monitoring tool. It's kinda what it does 😉

Go to a site's Site Information page by clicking on it in the home screen.

Scroll down to Extensions.

In the Filters line click the Missing Download ID filter icon (key).

Now you see which extensions are reported by Joomla as missing their Download IDs.

**Comment by PaulSrch** [comment]
Thanks for reminding me about the filter! I figured out which extension is the culprit - Latest News Enhanced Pro.

The notification is coming in the Panopticon sites list. The Joomla! Updates and Download Keys quick icons both show OK. and I re-entered the download key from my Simplify Your Web account. Still has the issue. Could it be something in the extension that's causing this?

I'm not really using the extension on that site, so it's not a big problem (I'll be removing the extension soon). But it is curious!

**Reply by nikosdion** [reply]
There might be a problem with the update sites.

If the update site is not connected to the correct extension, the extension will be reported as missing its download key.

If the extension has multiple update sites, it is possible that Joomla will not be picking up the correct update site and thus report the extension as missing its download key.
